main.notebook.py

```diff
@@ -18,19 +18,22 @@
 
 
 ## Motivation
-When trying to predict and output it is some times useful to also get a confidence score
-or similarly a range of values around this expected value in which the true value might be found. 
-Practical examples of this include estimating upper and lower bound when predicting a 
-time of arrival (ETA) or a stock price since you not only care about an expected value 
-but also about the best case and worst case scenarios when trying to minimize risk.
-
-While most Machine Learning techniques don't provide a natural way of doing this, 
-in this article we will be exploring **Quantile Regression** as a means of doing so, 
-this technique will allow us to learn some very important statistical properties 
+When trying to predict and output, it is sometimes helpful to get a confidence score
+or, similarly, a range of values around this expected value in which the actual value might be found. 
+Practical examples include estimating upper and lower bound when predicting a time of arrival (ETA) or a 
+stock price since you care about an expected value and the best case and worst-case scenarios when trying to minimize risk.
+
+While most Machine Learning techniques do not provide a natural way of doing this, 
+in this article, we will be exploring **Quantile Regression** as a means of doing so; 
+and this technique will allow us to learn some critical statistical properties 
 of our data: the quantiles.
 
-To begin our journey into quantile regression we will first get hold on some data:
+To begin our journey into quantile regression, we will first get a hold on some data, and install the necessary libraries.:
 """
+# %%
+! curl -Ls https://raw.githubusercontent.com/Davidnet/quantile-regression/master/requirements.txt > requirements.txt
```

Comment:
You can change to @cgarciae as soon as the requirements.txt are updated in main repo